src/bootstrap-table.js

```diff
@@ -721,17 +721,17 @@ class BootstrapTable {
       ) {
         opts[optionName] = true
       }
-
-      if (!opts.buttonsOrder.includes(buttonName)) {
+      if (opts.buttonsOrder.findIndex(item => buttonName.toLowerCase() === item.toLowerCase()) === -1) {
         opts.buttonsOrder.push(buttonName)
       }
     }
 
     // Adding the button html to the final toolbar html when the showOption is true
     for (const button of opts.buttonsOrder) {
-      const showOption = opts[`show${button.charAt(0).toUpperCase()}${button.substring(1)}`]
-
-      if (showOption) {
+      if (
+        opts[`show${button.charAt(0).toUpperCase()}${button.substring(1)}`] ||
+        opts[`show${button}`]
```

Comment:
This line seems not to work: https://live.bootstrap-table.com/code/wenzhixin/4840

Reply:
Your example works for me, i see the button.

Reply:
I mean to hide the button: `showbtnTest: false`.